Web/.eslintrc.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     'no-undef': 'off',
 
     'no-alert': 'off',
-    'prefer-destructuring': 'off',
+    'prefer-destructuring': 'error',
```

Comment:
```suggestion
```

Just remove it, seems to be default on by ESLint: https://eslint.org/docs/rules/prefer-destructuring